astropy/io/votable/tree.py

```diff
@@ -3505,6 +3511,18 @@ def _add_group(self, iterator, tag, data, config, pos):
         self.groups.append(group)
         group.parse(iterator, config)
 
+    def _get_version_flags(self):
```

Comment:
I'd maybe call this `_get_version_checks` or `_get_version_info`

Reply:
Done.